compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1302,7 +1302,6 @@ private STNode parseParameter(SyntaxKind prevParamKind, boolean isParamNameOptio
             case AT_TOKEN:
                 annots = parseOptionalAnnotations();
                 break;
-            case PUBLIC_KEYWORD:
```

Comment:
sorry I didn't see this earlier.  Have to remove parsing of public keyword in
https://github.com/ballerina-platform/ballerina-lang/pull/25228/files#diff-51d1fcc2878d0934de8afd17c12009dcL1336-L1338

Reply:
Will fix this!